packages/slate-react/src/plugins/after.js

```diff
@@ -511,7 +512,13 @@ function AfterPlugin(options = {}) {
         return editor.moveToStart()
       }
 
-      return editor.moveBackward()
+      const { start } = selection
```

Comment:
I think instead of inlining the length calculating logic here, we should instead make it handle a single "character" by default when no `n` is provided? (Instead of defaulting to `1`?)

Reply:
@ianstormtaylor To be clear, are you suggesting to move offset calculation into [moveForward/Backward](https://github.com/ianstormtaylor/slate/blob/master/packages/slate/src/commands/on-selection.js#L358)? Like it's already in [deleteCharForward/Backward/AtRange](https://github.com/ianstormtaylor/slate/blob/master/packages/slate/src/commands/at-range.js#L427)?

Reply:
@ianstormtaylor To be totally clear, are you suggesting to move offset calculation into [moveForward/Backward](https://github.com/ianstormtaylor/slate/blob/master/packages/slate/src/commands/on-selection.js#L358)? Like [deleteCharForward/Backward/AtRange](https://github.com/ianstormtaylor/slate/blob/master/packages/slate/src/commands/at-range.js#L427) already functions with the exception that only calculate it when no `length args` is provided.

Reply:
@adjourn Yup, moving it in there. But actually, maybe it's better to treat the `n` argument as "characters" rather than "length"? So that it's what you'd expect for all cases.